mediawiki/LocalSettings.d/LocalSettings.override.php

```diff
@@ -168,4 +167,5 @@
         'length' => 1000,
     ],
 ];
-$wgDefaultSkin = 'vector-2022';
```

Comment:
I think it should be vector-2022

Reply:
This would break the format (happens since updating to the new wmf branch 1.41), that's why I changed it back to simply vector

Reply:
Vector is ugly. Do you have a pointer to a broken instance. Locally you can follow https://www.mediawiki.org/wiki/Skin:Vector/2022

Reply:
I only have my local instance, which is the same you get if you run the compose file here. Even deactivating all the extensions vector-2022 breaks our system.

Reply:
Can you install vector-2022 on your system, manually to understand why it fails? I see no issues either on en.wikipedia.org or on portal.mardi4nfdi.de which both use the new vector version.

Reply:
We don't see any issues in portal.mardi4nfdi.de because it is still does not take into account this https://github.com/MaRDI4NFDI/docker-wikibase/commit/b5f497f6d493695208d334b7fd8cbb0471c6b64a